pyproject.toml

```diff
@@ -1,9 +1,10 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
 [project]
-name = "astrodb-scripts"
+name = "astrodb_scripts"
+version = "0.0.1"
```

Comment:
So, yes- remove version or set it to dynamic which I think means it will be handled by git tags.

Reply:
done!